Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:8-jre
```

Comment:
Kan vi oppgradere til siste java (15)? Se fdk-sparql-service for image navnet

Reply:
Lagde et issue: https://github.com/Informasjonsforvaltning/fdk-issue-tracker/issues/173